### PR TITLE
LPS-162959 buildLang

### PR DESCRIPTION
**Pull request by p2kmgcl**
The first three commits are from this PR to avoid conflicts: https://github.com/liferay-echo/liferay-portal/pull/9602

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/SpacingBox.js [hunk 1]
```diff
@@ -348,6 +351,59 @@ function SpacingOptionValue({
 	return value === undefined ? '' : value;
 }
 
+export function Tooltip({hoverElement, id: tooltipId, label, positionElement}) {
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/SpacingBox.js:354]: We don't need to export this anymore, right?

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/Tooltip.test.js [hunk 2]
```diff
@@ -1,82 +0,0 @@
-/**
```
Comment (sandrodw3) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/common/components/Tooltip.test.js:1]: Couldn't we move tests to `SpacingBox.test.js` so we don't lose them?

**Review by sandrodw3 — COMMENTED** [review]
Super cool! 🥳 just left two minicomments

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/PopoverTooltip.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+			className="position-fixed"
+			disableScroll
+			header={header}
+			id={id}
+			onShowChange={setShowPopover}
+			role="tooltip"
+			show={showPopover}
+			trigger={
+				<span
+					aria-describedby={id}
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/PopoverTooltip.js:42]: .

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by victorg1991** [comment]
https://github.com/liferay-echo/liferay-portal/pull/9658

**Pull request closed without merging**